scripts/parse_logs.py

```diff
@@ -293,6 +328,24 @@ def format_for_otel_fetch(group):
     }
 
 
+def format_for_otel_exception(group):
+    prefix = "exc"
+    exception = get_one(group["exception"])
+    start = exception["timestamp"]
+    end = start
+
+    return {
```

Comment:
so, in otel land, exceptions are usually added as events to a span, rather than spans in their own right, as they typically don't have a duration. And usually the span is marked as having errored, which highlights it in honeycomb.

However, we can take this format, as if its another event, and do that in the job-runner code?